docs/self-managed/platform-deployment/helm-kubernetes/guides/ingress-setup.md

```diff
@@ -191,21 +191,31 @@ Ingress resources require the cluster to have an [Ingress Controller](https://ku
 
 ### Example local configuration
 
-An Ingress Controller is also required when working a local Camunda 8 installation. Take a look at an Ingress Controller configuration using the [Nginx Ingress Controller](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/):
+An Ingress Controller is also required when working on a local Camunda 8 installation. Take a look at an Ingress Controller configuration using the [ingress-nginx Controller](https://kubernetes.github.io/ingress-nginx/deploy/#bare-metal-clusters/):
 
 ```yaml
-# nginx_ingress_values.yml
+# ingress_nginx_values.yml
 controller:
-  replicaCount: 1
-  hostNetwork: true
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
   service:
     type: NodePort
+
+  publishService:
+    enabled: false
 ```
 
-To install this Ingress Controller to your local cluster, execute the following command:
+To install this `ingress-nginx Controller` to your local cluster, execute the following command:
```

Comment:
Good point 👍

Reply:
yeah, wish there was an easier way to maintain the versioned docs ... it will only get more complicated for each version, especially infrastructure which is usually generally applicable independent of C8.